charts/velero/values.yaml

```diff
@@ -72,6 +72,12 @@ metrics:
 # Install CRDs as a templates. Enabled by default.
 installCRDs: true
 
+# Enable/disable helm hooks annotations for Helm v2 in CRDs and/or templates
+# You should disable this if using a deploy tool that doesn't support helm hooks
+enableHelmHooks:
+  crds: true
+  templates: true
```

Comment:
I'm thinking if we could use only one variable to cover all the scenarios?
For ArgoCD, what's the `enableHelmHooks.crds` and `enableHelmHooks.templates` would be?

Reply:
We could. At first I thought about use only one variable for all, like "enableHelmHooks: true", but I made it that way to cover any scenarios where disabling hooks on CRDs it's necessary but not on templates, although I don't have any in mind. 

ArgoCD has no problem with helm hooks on templates from v1.3, so it works with or without helm hooks on them.

One variable for all make it simpler, and is better, if there is really no use for keeping separated.

Reply:
@simran3695, @fabianmet could you please provide the ArgoCD version you used?
I can't reproduce #158 on ArgoCD version 1.7.

Reply:
> We could. At first I thought about use only one variable for all, like "enableHelmHooks: true", but I made it that way to cover any scenarios where disabling hooks on CRDs it's necessary but not on templates, although I don't have any in mind.
> 
> ArgoCD has no problem with helm hooks on templates from v1.3, so it works with or without helm hooks on them.
> 
> One variable for all make it simpler, and is better, if there is really no use for keeping separated.

I can't think of other scenarios, for now, but using a single variable makes it simpler to understand.

Reply:
> I can't think of other scenarios, for now, but using a single variable makes it simpler to understand.

Shall I change this to a single variable then? 

>@simran3695, @fabianmet could you please provide the ArgoCD version you used?
I can't reproduce #158 on ArgoCD version 1.7.

I saw the same problem that @simran3695 describe in #158 on version 1.5, where CRDs with PreSync keep syncing forever. Like @fabianmet [explained](https://github.com/vmware-tanzu/helm-charts/issues/158#issuecomment-692009835), it's a problem on how ArgoCD treats helm hooks, and when helm hooks are removed, all works fine.
For CRs like BSL and VSL I had no problems when using helm hooks/Post Sync on v1.5, but removing hooks also works fine (setting enableHelmHooks.templates: false)

Reply:
Maybe use a single variable would be easier:
```
enableHelmHooks: true
```

Reply:
Done! We can change this in the future if comes any scenario for that.